ltc build fails

### Background

The LTC docker image is outdated and has build errors.

### Your environment

docker

### Steps to reproduce

docker-compose build

### Expected behavior

should build without error

### Actual behavior

ltc images fails to build as follows:

```[ERROR] Failed to find glide.yaml file in directory tree: Cannot resolve parent of /
ERROR: Service 'ltc' failed to build: The command '/bin/sh -c apk add --no-cache git &&  git clone https://github.com/ltcsuite/ltcd ./ &&  go get -u github.com/Masterminds/glide &&  glide install &&  go install . ./cmd/ltcctl ./cmd/gencerts' returned a non-zero code: 2```

--- a/docker/ltcd/Dockerfile
+++ b/docker/ltcd/Dockerfile
@@ -1,15 +1,14 @@
-FROM golang:1.10-alpine as builder
+FROM golang:1.11-alpine as builder
 
 MAINTAINER Olaoluwa Osuntokun <lightning.engineering>
 
 # Grab and install the latest version of roasbeef's fork of ltcd and all
 # related dependencies.
 WORKDIR $GOPATH/src/github.com/ltcsuite/ltcd
-RUN apk add --no-cache git \
-&&  git clone https://github.com/ltcsuite/ltcd ./ \
-&&  go get -u github.com/Masterminds/glide \
-&&  glide install \
-&&  go install . ./cmd/ltcctl ./cmd/gencerts
+RUN apk add --no-cache --update alpine-sdk git
+RUN git clone https://github.com/ltcsuite/ltcd ./
+RUN GO111MODULE=on go install -v . ./cmd/...
+RUN GO111MODULE=on go install . ./cmd/ltcctl ./cmd/gencerts
 
 # Start a new image
 FROM alpine as final

